controller/workitemtype.go

```diff
@@ -72,15 +72,17 @@ func (c *WorkitemtypeController) Create(ctx *app.CreateWorkitemtypeContext) erro
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
-		witTypeModel, err := appl.WorkItemTypes().Create(
+		witTypeModel, err := appl.WorkItemTypes().CreateWithDates(
 			ctx.Context,
 			*ctx.Payload.Data.Relationships.Space.Data.ID,
 			ctx.Payload.Data.ID,
 			ctx.Payload.Data.Attributes.ExtendedTypeName,
 			ctx.Payload.Data.Attributes.Name,
 			ctx.Payload.Data.Attributes.Description,
 			ctx.Payload.Data.Attributes.Icon,
-			modelFields)
+			modelFields,
+			ctx.Payload.Data.Attributes.CreatedAt,
```

Comment:
This is used in the testing but I can see that it might be dangerous to allow the client to set  creation or updated date. Will revise that. Good catch.